src/main/java/com/iemr/common/data/beneficiary/Beneficiary.java

```diff
@@ -178,7 +179,7 @@ public class Beneficiary
 
 	/**
 	 * @param beneficiaryRegID
-	 *            the beneficiaryRegID to set
+	 *                         the beneficiaryRegID to set
```

Comment:
indent of these comments look off

Reply:
I was using an extension of VS code called Prettier due to which it happened. Now, I have changed it.